### PR TITLE
Fibery Integration

### PR DESCRIPTION
**Pull request by Drapegnik**
- fibery renderer
- add preview feature
- audio & video article page

https://github.com/babajka/babajka-backend/pull/55

### CODE REVIEW

**Comment by Drapegnik** [comment]
[fibery integration [articles]](https://trello.com/c/77KzjNrk/380-fibery-integration-articles)

**Comment by Drapegnik** [comment]
[fibery rich content renderer](https://trello.com/c/Hwn19iB0/386-fibery-rich-content-renderer)

**Comment by Drapegnik** [comment]
[add article preview page](https://trello.com/c/3CQ6Ayy8/389-add-article-preview-page)

File: components/articles/Article.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+      <MetaDescription description={subtitle} />
+      {/* FIXME */}
+      <MetaImage url={images.page} />
+      <MetaKeywords keywords={keywords} />
+      <div className="article-page">
+        <div className="article-page-margins">{subtitle}</div>
+        {type === 'text' && <img className="article-page__cover" src={images.page} alt={title} />}
+      </div>
+      <div className="article-page-margins article-page__header">
+        {/* TODO: hover corresponding image and title simulteneously */}
```
Comment (UladBohdan) [inline components/articles/Article.js:47]: simultAneously (sorry)

File: components/articles/cards/article/CollectionCard.js [hunk 2]
```diff
@@ -5,15 +5,15 @@ import React from 'react';
 import Text from 'components/common/Text';
 
 const CollectionCard = ({
-  covers,
+  images,
```
Comment (UladBohdan) [inline components/articles/cards/article/CollectionCard.js:8]: `covers` became `images`, `image` became a `cover` 😂

**Review by UladBohdan — APPROVED** [review]

**Pull request merged**